assets/components/src/button-card/style.scss

```diff
@@ -17,8 +17,20 @@
 	transition: background-color 125ms ease-in-out, border-color 125ms ease-in-out,
 		box-shadow 125ms ease-in-out;
 
-	&:active,
 	&:focus,
+	&.is-pressed {
+		background: white;
```

Comment:
It seems like the ButtonCard component is only used in the admin, so this _should_ be fine—is there any scenario where the the `white` might clash with another color that's configurable?

Reply:
At this point, not that I can think of.